# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/handle_closer.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/handle_closer.h
--- a/security/sandbox/chromium/sandbox/win/src/handle_closer.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/handle_closer.h	Wed Jun 12 11:10:48 2019 +0100
@@ -21,19 +21,19 @@
 // This is a map of handle-types to names that we need to close in the
 // target process. A null set means we need to close all handles of the
 // given type.
-typedef std::map<const base::string16, std::set<base::string16> > HandleMap;
+typedef std::map<const base::string16, std::set<base::string16>> HandleMap;
 
 // Type and set of corresponding handle names to close.
 struct HandleListEntry {
-  size_t record_bytes;       // Rounded to sizeof(size_t) bytes.
-  size_t offset_to_names;    // Nul terminated strings of name_count names.
+  size_t record_bytes;     // Rounded to sizeof(size_t) bytes.
+  size_t offset_to_names;  // Nul terminated strings of name_count names.
   size_t name_count;
   base::char16 handle_type[1];
 };
 
 // Global parameters and a pointer to the list of entries.
 struct HandleCloserInfo {
-  size_t record_bytes;       // Rounded to sizeof(size_t) bytes.
+  size_t record_bytes;  // Rounded to sizeof(size_t) bytes.
   size_t num_handle_types;
   struct HandleListEntry handle_entries[1];
 };
@@ -47,8 +47,8 @@
   ~HandleCloser();
 
   // Adds a handle that will be closed in the target process after lockdown.
-  // A NULL value for handle_name indicates all handles of the specified type.
-  // An empty string for handle_name indicates the handle is unnamed.
+  // A nullptr value for handle_name indicates all handles of the specified
+  // type. An empty string for handle_name indicates the handle is unnamed.
   ResultCode AddHandle(const base::char16* handle_type,
                        const base::char16* handle_name);
 